src/lib/hooks/useConnect/connect.ts

```diff
@@ -0,0 +1,38 @@
+import { disconnectSdk } from './disconnectSdk';
+import { setUpConnection } from './setupConnection';
+import { ConnectTypes } from './types';
+
+export async function connect({
+  logger,
+  sdkDispatcher,
+  userDispatcher,
+  userId,
+  appId,
+  customApiHost,
+  customWebSocketHost,
+  configureSession,
+  nickname,
+  profileUrl,
+  accessToken,
+  sdk,
+}: ConnectTypes): Promise<void> {
+  await disconnectSdk({
+    logger,
+    sdkDispatcher,
+    userDispatcher,
+    sdk,
+  });
+  await setUpConnection({
```

Comment:
Should we wrap here \w try - catch just in case disconnectSdk or setUpConnection throws an error?

Reply:
Before, we used to throw it to customer, can consider wrapping, yeah

Reply:
Ah, actually wrapped in `useConnect` ~